verticapy/performance/vertica/qprof.py

```diff
@@ -1572,6 +1577,7 @@ def _check_v_table(self, iterchecks: bool = True) -> None:
                                 transaction_id = {tr_id}
                             AND statement_id = {st_id}
                             LIMIT 1"""
+                        query = self._replace_schema_in_query(query)
```

Comment:
@oualib There is an error. I think this line is causing the schema name and key id to be duplicated in the query.

Reply:
Please remove them, there are two.